packages/react-components/src/components/MessageThread.tsx

```diff
@@ -244,7 +244,7 @@ const ParticipantSystemMessageComponent = ({
   if (participantsStr !== '') {
     return (
       <SystemMessageComponent
-        iconName={(message.iconName ?? '') as SystemMessageIconTypes}
```

Comment:
This could be a bug of babel parser - a combination of ?? and type cast. This is super rare and we might also want to deal with prop type ourselves to avoid type cast anyway